tools/Evm/Evm/t8n/T8NInputProcessor.cs

```diff
@@ -0,0 +1,101 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Ethereum.Test.Base;
+using Evm.t8n.Errors;
+using Evm.t8n.JsonTypes;
+using Nethermind.Core.Specs;
+using Nethermind.Evm.Tracing.GethStyle;
+using Nethermind.Int256;
+using Nethermind.Serialization.Rlp;
+using Nethermind.Specs;
+using Nethermind.Specs.Forks;
+using Nethermind.Specs.Test;
+
+namespace Evm.t8n;
+
+public static class T8NInputProcessor
+{
+    private static readonly TxDecoder TxDecoder = TxDecoder.Instance;
```

Comment:
We can now potentially register transaction types (including RLP and JSON parsing) from plugins. I expect this won't be able to generate tests around those injected tx types. Maybe worth to consider adding plugins support.